css-pseudo-classes/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>CSS Pseudo Classes</title>
+  </head>
+
+  <body>
+    <link rel="stylesheet" href="styles.css" />
```

Comment:
This is already inside your `head` on line 7, so this should be deleted. Your `link` tags will always go inside the `head`